pkg/version/version.go

```diff
@@ -16,7 +16,7 @@ var Version = undefinedVersion
 // https://github.com/linkerd/linkerd2-proxy-init
 // This has to be kept in sync with the constraint version for
 // github.com/linkerd/linkerd2-proxy-init in /Gopkg.toml
-var ProxyInitVersion = "v1.0.0"
+var ProxyInitVersion = "87ab4bd22a6e45ae74ff05efca7c7bed5b5a5cce"
```

Comment:
This will break installations without `--linkerd-cni-enabled` because there's no init container image published with this tag. I think we should leave it to `v1.0.0` while this PR gets tested, and after the changes in the `linkerd2-proxy-init` repo get published under a new version tag we can set it here, although that wouldn't be strictly necessary since those changes only apply to the CNI script, not to the init containers.

Reply:
Released [linkerd2/proxy-init v1.1.0](https://github.com/linkerd/linkerd2-proxy-init/releases) to address this.